modules/light-clients/08-wasm/types/proposal_handle_test.go

```diff
@@ -78,203 +78,6 @@ func (suite *TypesTestSuite) TestCheckSubstituteAndUpdateStateGrandpa() {
 	}
 }
 
-// func (suite *TypesTestSuite) TestCheckSubstituteAndUpdateStateBasicTendermint() {
```

Comment:
this test is covered in [this pr](https://github.com/cosmos/ibc-go/pull/4950/files)